src/lib.rs

```diff
@@ -138,3 +188,16 @@ enum Reactor {
     Handle(reactor::Handle),
     Remote(reactor::Remote),
 }
+
+cfg_if! {
+    if #[cfg(feature = "tls")] {
+        extern crate tokio_tls;
+        extern crate native_tls;
+
+        /// Re-exported TLS-related types
+        pub mod tls {
+            pub use native_tls::Error as NativeTlsError;
```

Comment:
Done. Used `extern crate native_tls as native_tls_inner` to avoid the name spacing problem.